### PR TITLE
Feature/ussd nurse rapidpro

### PR DESCRIPTION
**Pull request by OMosimege**
Removed identity store
Added function normalising msisdns
Added background WhatsApp check
Added facility code validation function, whatsapp check, and jembi api call function

### CODE REVIEW

File: go-app-ussd_nurse_rapidpro.js [hunk 1]
```diff
@@ -2,33 +2,110 @@ var go = {};
...
+    var utils = SeedJsboxUtils.utils;
     var App = vumigo.App;
 
     var GoNDOH = App.extend(function(self) {
         App.call(self, 'state_start');
-        var utils = SeedJsboxUtils.utils; //replace with utils not in Seed
 
         //variables for services
-        var is;
+        /*var is;*/
```
Comment (erikh360) [inline go-app-ussd_nurse_rapidpro.js:23]: Can we remove this instead of commenting it out?

File: go-app-ussd_nurse_rapidpro.js [hunk 2]
```diff
@@ -2,33 +2,110 @@ var go = {};
...
 
         self.init = function() {
             // initialise services
             //replace identity store
-            is = new IdentityStore(
+            /*is = new IdentityStore(
```
Comment (erikh360) [inline go-app-ussd_nurse_rapidpro.js:28]: Can we remove this instead of commenting it out?

**Review by erikh360 — COMMENTED** [review]
Just the two comments. Nice!

File: go-app-ussd_nurse_rapidpro.js (hunk 1 above)
Comment (OMosimege) [reply go-app-ussd_nurse_rapidpro.js:23]: Oh thanks for pointing this out

File: go-app-ussd_nurse_rapidpro.js (hunk 2 above)
Comment (OMosimege) [reply go-app-ussd_nurse_rapidpro.js:28]: ahh this too thanks

**Review by erikh360 — APPROVED** [review]

**Pull request merged**